contracts/oracles/lido/LidoOracle.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import '@yield-protocol/utils-v2/contracts/access/AccessControl.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastBytes32Bytes6.sol';
+import '@yield-protocol/utils-v2/contracts/token/IERC20Metadata.sol';
+import '@yield-protocol/vault-interfaces/IOracle.sol';
+import '../../constants/Constants.sol';
+import './IWstETH.sol';
+
+/**
+ * @title LidoOracle
+ * @notice
+ */
+contract LidoOracle is IOracle, AccessControl, Constants {
+    using CastBytes32Bytes6 for bytes32;
+    IWstETH wstETH;
+    bytes32 public immutable wstEthId;
+    bytes32 public immutable stEthId;
+
+    constructor(bytes32 wstEthId_, bytes32 stEthId_) public {
```

Comment:
I ran Slither on this contract and the only hit was this:
```
Warning: Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
  --> contracts/oracles/lido/LidoOracle.sol:21:5:
   |
21 |     constructor(bytes32 wstEthId_, bytes32 stEthId_) public {
   |     ^ (Relevant source part starts here and spans across multiple lines).


```

Reply:
Ah yeah, `public` was only used in constructors before solidity 0.8